docs/connectors/index.mdx

```diff
@@ -109,132 +118,49 @@ for an example of configuring the output buffer.
-          "config": {}
-      }
-  }
-}'
-```
-
-#### Delete connector
-
-Delete connector named `product-tools`.
-
-```bash
-curl -i -X DELETE http://localhost:8080/v0/connectors/product-tools
-```
-
-#### Attach input and output connector
-
-A Feldera pipeline consists of an SQL program with input and output connectors attached to it.
-The user lists connectors to be attached to the pipeline as part of the pipeline configuration.
-In this example, we create pipeline `supply-chain-pipeline` with program `supply-chain-program`.
-The pipeline attaches input connector `product-tools` to table `product`
-and output connector `kafka-average-price` to view `average_price`.
-
-```
-curl -i -X PUT http://localhost:8080/v0/pipelines/supply-chain-pipeline \
--H 'Content-Type: application/json' \
--d '{
-    "description": "Supply Chain pipeline",
-    "program_name": "supply-chain-program",
-    "config": {"workers": 4},
-    "connectors": [
-         {
-             "connector_name": "product-tools",
-             "is_input": true,
-             "name": "product-tools",
-             "relation_name": "product"
-         },
-         {
-             "connector_name": "kafka-average-price",
-             "is_input": false,
-             "name": "kafka-average-price",
-             "relation_name": "average_price"
-         }
-    ]
-}'
-```
-
-### Python (direct API calls)
-
-**Retrieve list of connectors while providing authorization:**
-```python
-import requests
-
-api_url = "http://localhost:8080"
-headers = { "authorization": f"Bearer <API-KEY>" }
-
-for connector in requests.get(f"{api_url}/v0/connectors", headers=headers).json():
-    print(connector)
+A program with a view using an output connector that sends changes to a `test_view` Kafka topic in a
+JSON format suitable for consumption by Debezium:
+
+```sql
+create table test_table(
+    id bigint not null primary key,
+    f1 boolean,
+    f2 string,
+    f3 tinyint,
+    f4 decimal(5,2),
+    f5 float64,
+    f6 time,
+    f7 timestamp,
+    f8 date,
+    f9 binary
+);
+
+create view test_view
+WITH (
+    'connectors' = '[{
+        "max_queued_records": 1000,
+        "format": {
+            "name": "json",
+            "config": {
+                "update_format": "debezium"
+            }
+        },
+        "transport": {
+            "name": "kafka_output",
+            "config": {
+                "bootstrap.servers": "redpanda:9092",
+                "topic": "test_view"
+            }
+        }
+    }]'
+)
+as select * from test_table;
 ```
 
 ## Additional resources
 
 For more information, see:
 
-* [REST API curl tutorial](/docs/tutorials/rest_api)
-
-* [REST API documentation](https://www.feldera.com/api/create-a-new-connector) for connectors.
+* [Tutorial using HTTP-based Input and Output](/docs/tutorials/basics/part2)
+* [Supported transport for Tables/Sources](/docs/connectors/sources)
+* [Supported transport for Views/Sinks](/docs/connectors/sinks)
```

Comment:
* [Supported data formats]()
// There's currently no top-level page for formats that can be referenced here. Can we add it?